lib/file_management.js

```diff
@@ -0,0 +1,124 @@
+const archethic = require('archethic')
+const chalk = require('chalk')
+const crypto = require('crypto')
+const fs = require('fs')
+const path = require('path')
+const jsdom = require("jsdom")
+const {
+    JSDOM
+} = jsdom
+const mime = require('mime')
+
+function confirmationCallback(endpoint, address, message, file) {
```

Comment:
I don't think this should go here, it's not related about file but transaction handling